package.json

```diff
@@ -18,28 +18,34 @@
     "utils-merge": "1.0.1"
   },
   "devDependencies": {
+    "@types/node": "^10.12.18",
```

Comment:
So this is just a general question: will this cause issues for folks using this module with other Node.js versions like the 8.x series? If the types are for the 10.x series?

Reply:
Its just for the typings on the test. It won't explode downstream.